src/development/platform-integration/desktop.md

```diff
@@ -57,7 +57,8 @@ you need the following software:
 For Windows desktop development,
 you need the following in addition to the Flutter SDK:
 
-* [Visual Studio 2022][] When installing Visual Studio,
+* [Visual Studio 2022][] or [Visual Studio Build Tools 2022][]
+  When installing Visual Studio or only the Build Tools,
```

Comment:
Since most users will be more comfortable with a graphical IDE to go with their install, and because it's analogous to what we recommend for macOS (a full Xcode install), I'd phrase this something like:

* [Visual Studio 2022][] When installing Visual Studio,
  select the "Desktop development with C++" workload,
  including all of its default components,
  to install the necessary C++ toolchain and
  Windows SDK header files. Alternatively, 
  [Visual Studio Build Tools 2022][] can be installed if
  only the Windows command-line tools are desired.

Reply:
This sounds like you don't need to install the C++ workload with Build tools in order to be able to develop in windows.
My proposal:
```
Alternatively, [Visual Studio Build Tools 2022][] 
can be installed with the same workload if
only the Windows command-line tools are desired.
```

Reply:
@cbracken any opinions? Thx :)